docs/README.linux

```diff
@@ -101,19 +101,16 @@ Tip: By adding -j<number> to the make command, you describe how many
      concurrent jobs will be used. So for dualcore the command is:
 
     $ make -j2
- 	
+
 .3  $ make install
 
 This will install Kodi in the prefix provided in 4.1 as well as a launcher script.
 
-Note: You may need to run this with sudo (sudo make install) if your user doesn't have write permissions
-to the prefix you have provided (as in the default case, /usr/local).
-
 Tip: To override the location that Kodi is installed, use PREFIX=<path>.
 For example.
 
-    $ make install DESTDIR=$HOME/kodi
-	
+    $ sudo make install DESTDIR=$HOME/kodi
```

Comment:
make install generally requires sudo since generally its system dirs, that particular example doesnt, it makes sense to have it anyway, I rather change the example though since I removed the block of txt above which was commited in previous commit so you cant see the history in this one..